src/NodaTime/Period.cs

```diff
@@ -831,7 +844,7 @@ public Duration ToDuration()
         /// changes made to the builder are not reflected in this period.
         /// </summary>
         /// <returns>A builder with the same values and units as this period.</returns>
-        [Pure] [TestExemption(TestExemptionCategory.ConversionName)] public PeriodBuilder ToBuilder() => new PeriodBuilder(this);
+        [Pure][TestExemption(TestExemptionCategory.ConversionName)] public PeriodBuilder ToBuilder() => new PeriodBuilder(this);
```

Comment:
@jskeet apologies for the formatting change. I might not have set up my editor correctly. Is there an .editorconfig or similar that I should work with?

Reply:
No, I don't have an editorconfig for this repo at the moment I'm afraid. Can you just revert this change and the one below manually?
(And you can revert the change to PeriodPattern.cs completely.)

Reply:
Done. I'm generally careful about these things before I push a commit but I guess it just slipped my mind.